airbyte-integrations/connectors/source-zendesk-support/metadata.yaml

```diff
@@ -26,4 +26,9 @@ data:
     sl: 300
     ql: 400
   supportLevel: certified
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "`cursor_field` for `Tickets` stream is changed to `generated_timestamp`"
+        upgradeDeadline: "2023-07-19"
```

Comment:
This is a poor breaking change message.  Why was this change?  What do users need to do to get syncs working again?

Reply:
Are there any docs on how to upgrade to this version?